README.md

```diff
@@ -31,6 +34,16 @@ The Library supports all APIs under the following services:
 * https://docs.adyen.com/developers/development-resources/libraries
 * https://docs.adyen.com/developers/checkout/api-integration
 
+## HTTP Client Configuration
+
+In order to submit http requests to adyen api you need to initialize the client. The following example makes a checkout payment request:
+
+```csharp
+var client = new Client("YOUR-XAPI-KEY", Model.Enum.Environment.Test); //or Model.Enum.Environment.Live
+var checkout = new Checkout(client);
+var paymentsResponse = checkout.Payments(paymentsRequest);
```

Comment:
Could you add a simple payment request here ? Let's use the format described internally check our internal documentation.

Reply:
@rikterbeek Can you check again ?